lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -270,7 +270,7 @@ def answer_dsl_module
 
                   define_method("#{key}=") do |v|
                     self.value ||= Hash.new
-                    self.value[key] = v
+                    self.value[key] = v.andand.strip
```

Comment:
Oke, prima. Dan is het idd beter om op een later tijdstip de serverkant te versimpelen.